src/util/invariant.h

```diff
@@ -230,8 +236,9 @@ CBMC_NORETURN
   abort();
 }
 
-/// Takes a backtrace, constructs an invariant_violatedt from reason and the
-/// backtrace, aborts printing the invariant's description.
+/// This is a wrapper class used by the macro 'INVARIANT(CONDITION, REASON)'
```

Comment:
Again, I am wondering whether this is in the right place - it's documenting a method, but talks about a class?

Reply:
Argh! Fixed.